### DISCUSSION TITLE
lMS

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#1338] [Feature Requests]
Would it be possible to use lMS natively instead of MPD ?
Would be great to easily connect QoBuz for ex.

Raudio based on arch is (I think) the best audio streamer software running with a raspberry.
It's better than picore but LMS propose more versatile user features .

**Comment by NexusPenguin** [comment]
Hi,

Well, I've tried LMS. 
I did not liked it. 

But maybe I have overlooked some killer fatures. 

What are those "versatile features you're talking about ?

Regards. 

FRED

**Reply by Rasb74** [reply]
Hi
Like you the first time I used LMS I don't like it.
But i decided to go deeper with it.
Huge plug-in offer , as tidal and QoBuz connect for ex
Song lyrics fully working
Performers stories working
Album info working
Multi room system very easy to use and manage
...
To use it on ipad/iPhone I purchased for few penny's IPeng app, with this app the system is easier to use and better looking !

**Reply by NexusPenguin** [reply]
Hi,

Indeed, looks quite interesting. 

So all this exists in piCorePlayer ? 

I'll give it another try, but I really was not able to do anything with it...

Regards.
